pyproject.toml

```
@@ -33,6 +32,7 @@ dynamic = ['version']
 test = [
     'flake8',
     'pytest',
+    'pandas',
```

```suggestion
    'ax-platform >= 0.3.4',
```
We need Ax to run the tests